app/src/main/java/io/intrepid/contest/screens/contestcreation/editcategoriestocontest/EditCategoriesFragment.java

```diff
@@ -23,6 +25,14 @@
     EditText categoryDescriptionField;
     private ActivityCallback activity;
 
+    public static Fragment newInstance(@Nullable Category category) {
+        EditCategoriesFragment fragment = new EditCategoriesFragment();
+        Bundle args = new Bundle();
+        args.putParcelable(CATEGORY_KEY, category);
+        fragment.setArguments(args);
```

Comment:
As we talked, I was a little confused about a fragment setting arguments for itself to use, but maybe I'm just not used to this way of keeping data during multiple fragments' lifecycle.